exporters/otlp/otlplog/otlplogfile/exporter.go

```diff
@@ -0,0 +1,95 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package otlplogfile // import "go.opentelemetry.io/otel/exporters/otlp/otlplog/otlplogfile"
+
+import (
+	"context"
+	"sync"
+
+	"google.golang.org/protobuf/encoding/protojson"
+
+	"go.opentelemetry.io/otel/exporters/otlp/otlplog/otlplogfile/internal/transform"
+	"go.opentelemetry.io/otel/exporters/otlp/otlplog/otlplogfile/internal/writer"
+	"go.opentelemetry.io/otel/sdk/log"
+	lpb "go.opentelemetry.io/proto/otlp/logs/v1"
+)
+
+// Exporter is an OpenTelemetry log exporter that outputs log records
+// into JSON files. The implementation is based on the specification
+// defined here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/protocol/file-exporter.md
```

Comment:
This should be a permalink to clarify which version of the spec we currently support.

```suggestion
// defined here: https://github.com/open-telemetry/opentelemetry-specification/blob/v1.36.0/specification/protocol/file-exporter.md
```